### PR TITLE
A usage statement

### PR DESCRIPTION
**Pull request by kirlat**
Added a usage statement. Removed default parameters. Will print a usage statement when run with incorrect number of arguments.

### CODE REVIEW

File: dist/build.mjs [hunk 1]
```diff
@@ -60,6 +60,31 @@ class Build {
     }
   }
 
+  static checkArgs () {
+    console.log('check args')
+    if (process.argv.length <= 5) {
```
Comment (balmas) [inline dist/build.mjs]: shouldn't this test be for < 5 rather than <= 5?

**Review by balmas — CHANGES_REQUESTED** [review]
Additional bug I think: in get webpackModule it should return Build.modules[Build.modules.length-1] rather than     return Build.modules[Build.modules.length] (running with webpack as the module fails as currently coded)

File: dist/build.mjs (hunk 1 above)
Comment (kirlat) [reply dist/build.mjs]: Yes, will fix it

**Comment by kirlat** [comment]
Thanks for catching errors, submitted fixes

**Review by balmas — APPROVED** [review]
Looks good!

**Pull request closed without merging**